lib/graphqr/base_resolvers.rb

```diff
@@ -0,0 +1,40 @@
+# frozen_string_literal: true
+
+module GraphQR
+  ##
+  # @TODO doc
+  module BaseResolvers
+    ##
+    # @TODO doc
+    def base_collection_resolver(type_class, scope_class)
+      Class.new(::BaseResolver) do
```

Comment:
this class is inheriting from a `BaseResolver` that comes from the user
if the user does not includes `ApplyScopes` and `AuthorizeGraphQL` it will break

we could:
1. Define our own `GraphQR::BaseResolver`
2. Delete the inheritance from this class and include the modules manually

I think option 1 is better, what do you think?

Reply:
The 1st one make complete sense,
i'll go with it